test/conformance/kuttl/ttl/past-timestamp/04-past-timestamp.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: test-pod-2
+  labels:
+    cleanup.kyverno.io/ttl: 2023-07-19T120000Z
```

Comment:
Please put the resource in a separate yaml file and use a TestStep

Reply:
Fixed